<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="sonali" failures="1" tests="3" name="IncludeExcludeClass1and2(failed)(failed)(failed)" time="0.066" errors="0" timestamp="2024-01-02T16:38:45 IST">
  <testcase classname="TestNGStudy.TestClass1" name="Tc5" time="0.008"/>
  <testcase classname="TestNGStudy.TestClass1" name="Tc2" time="0.003">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNGStudy.TestClass1.Tc2(TestClass1.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Tc2 -->
  <testcase classname="TestNGStudy.TestClass2" name="Tc6" time="0.002"/>
  <testcase classname="TestNGStudy.TestClass2" name="Tc4" time="0.0">
    <ignored/>
  </testcase> <!-- Tc4 -->
  <testcase classname="TestNGStudy.TestClass1" name="Tc1" time="0.0">
    <ignored/>
  </testcase> <!-- Tc1 -->
  <testcase classname="TestNGStudy.TestClass1" name="Tc3" time="0.0">
    <ignored/>
  </testcase> <!-- Tc3 -->
  <testcase classname="TestNGStudy.TestClass1" name="Tc4" time="0.0">
    <ignored/>
  </testcase> <!-- Tc4 -->
</testsuite> <!-- IncludeExcludeClass1and2(failed)(failed)(failed) -->
